controllers/periodic_healthcheck.go

```diff
@@ -107,6 +107,10 @@ func (r *EtcdadmClusterReconciler) periodicEtcdMembersHealthCheck(ctx context.Co
 	currClusterHFConfig := etcdadmClusterMapper[etcdCluster.UID]
 	endpoints := strings.Split(etcdCluster.Status.Endpoints, ",")
 	for _, endpoint := range endpoints {
+		if len(endpoint) == 0 {
```

Comment:
if the `etcdCluster.Status.Endpoints` field was empty, the for loop iteration shouldn't have started, maybe we need this check earlier